lookout/style/format/default_config.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+DEFAULT_STYLE_ANALYZER_CONFIG = {
+    "common": {
+        # All common for train and analyze config values for any language should be here
+        "comment_template": os.path.join(os.path.dirname(__file__), "templates", "comment.jinja2"),
```

Comment:
No "aggregate" here but we are using it above

Reply:
it is for quality report analyzer. The default value is defined here: https://github.com/src-d/style-analyzer/pull/650/files#diff-17b9445600e10a2c232356a5f1831885R213